wall_e/src/resources/locales/help.json

```diff
@@ -145,7 +145,13 @@
 			"access": "roles",
 			"roles" : [ "@everyone" ],
 			"name": [ "remindmein" ],
```

Comment:
you forgot to add the alias here.

Reply:
Won't be able to work on it for a little bit but I'll get that done by tonight

Reply:
Done now
I also aliased it to .remindmeat for better grammar, e.g. ".remindmeat 6:55pm to submit assignment"